index.bs

```diff
@@ -869,6 +883,23 @@ To <dfn>press keys</dfn> given |command parameters|:
 
 </div>
 
+<div algorithm>
+
+Note: Remote ends which do not implement [=press keys=] might choose to
+implement other intents to facilitate navigation. Such implementations could
+possibly meet clients' needs for traversal and basic interaction, but they
+would be unable to facilitate the filling of form fields. For clients of such
+implementations who are targeting web applications, the following intent could
+be used in conjunction with the keyboard simulation faculties of [[WEBDRIVER]]
+or [[WEBDRIVER-BIDI]] in order to meet such needs.
+
+To <dfn>activate element</dfn> given <var ignore>command parameters</var>:
+
+1. Perform the default action for the item in the [=virtual cursor=].
+1. Return [=success=] with data null.
```

Comment:
@jugglinmike Should this be 

```suggestion
2. Return [=success=] with data null.
```

Reply:
Yes, good catch!

(Technically, it doesn't matter to the Markdown parser, so a lot of specs will use the number "1" everywhere because that reduces diff noise when adding/removing steps. I can't remember why we're using accurate numbering throughout AT Driver.)